agent/azure-monitor-exporter/src/main/java/com/azure/monitor/opentelemetry/exporter/implementation/pipeline/TelemetryPipeline.java

```diff
@@ -154,4 +165,24 @@ private void onResponseBody(
       result.fail();
     }
   }
+
+  private void manageDailyQuota(HttpResponse response) {
+    if (isNewDailyQuotaExceeded(response)) {
```

Comment:
```suggestion
    if (response.getStatusCode() == 402) {
```

Reply:
The method `isNewDailyQuotaExceeded` may give meaning to `response.getStatusCode() == 402`. When we read the code with the method, we quickly see that the condition is for the business use case "new daily quota exceeded". The code may be more explicit?